File: front-end/src/app/service-models/app/v1/client/dash-board-s-m.ts
import { ClientCompanyDepartmentReportSM } from './client-company-department-report-s-m';

export class DashBoardSM {
    numberOfEmployees!: number;
    numberOfAdmins!: number;
    numberOfLeavesApproved!: number;
    numberOfLeavesPending!: number;
    numberOfLeavesRejected!: number;
    numberOfDepartments!: number;
    numberOfEmployeesPresent!: number;
    numberOfEmployeesAbsent!: number;
    numberOfEmployeeOnLeave!: number;
    clientCompanyDepartment!: Array<ClientCompanyDepartmentReportSM>;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/Client/DashBoardSM.cs
using System.Collections.Generic;

namespace Siffrum.Web.Payroll.ServiceModels.v1.Client
{
    public class DashBoardSM
    {
        public int NumberOfEmployees { get; set; }
        public int NumberOfAdmins { get; set; }
        public int NumberOfLeavesApproved { get; set; }
        public int NumberOfLeavesPending { get; set; }
        public int NumberOfLeavesRejected { get; set; }
        public int NumberOfDepartments { get; set; }
        public int NumberOfEmployeesPresent { get; set; }
        public int NumberOfEmployeesAbsent { get; set; }
        public int NumberOfEmployeeOnLeave { get; set; }
        public List<ClientCompanyDepartmentReportSM> ClientCompanyDepartment { get; set; }
    }
}
